src/client/common/types.ts

```diff
@@ -186,7 +186,7 @@ export interface IWatchableJupyterSettings extends IJupyterSettings {
     readonly onDidChange: Event<void>;
 }
 
-export type LoggingLevelSettingType = 'off' | 'error' | 'warn' | 'info' | 'debug';
+export type LoggingLevelSettingType = 'off' | 'error' | 'warn' | 'info' | 'debug' | 'verbose';
```

Comment:
I personally think we should clean all of this, I don't know if anyone uses them. Feels rather redundant to have `warn|info|debug`, I think `off|error|verbose` are sufficient.

Reply:
Something for debt week